### DISCUSSION TITLE
20.6.2

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1189] [General]
## Releases Notes for 20.6.2
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/draw.io-20.6.2-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/draw.io-20.6.2-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/draw.io-universal-20.6.2.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/drawio-amd64-20.6.2.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/drawio-x86_64-20.6.2.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v20.6.2/drawio-x86_64-20.6.2.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Fixes startup issues seen on some windows installs
- Updates to [draw.io core 20.6.2](https://github.com/jgraph/drawio/blob/v20.6.2/ChangeLog). All changes from 20.6.2 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v20.6.2'>20.6.2</a>.</em>

**Comment by jelovac** [comment]
Will there be an update for the snap version as it is still 20.6.0 on the snapcraft website?

**Comment by dmalawey** [comment]
### Question
Could you briefly describe advantages of install versus non-install version?

**Reply by dmnsea** [reply]
@dmalawey Non-install version is like usual portable (except settings are stored somewhere in user's folder), meaning that you can just give non-install version to anybody else and it will be ready to use.
At least that's what I see.
